app/src/main/java/com/belatrixsf/allstars/adapters/EmployeeListAdapter.java

```diff
@@ -74,14 +78,25 @@ public void updateData(List<Employee> employees){
         notifyDataSetChanged();
     }
 
-    static class EmployeeViewHolder extends RecyclerView.ViewHolder{
+    static class EmployeeViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+
         @Bind(R.id.photo) public ImageView photo;
         @Bind(R.id.full_name) public TextView fullName;
         @Bind(R.id.level) public TextView level;
+        private RecyclerOnItemClickListener recyclerOnItemClickListener;
 
-        public EmployeeViewHolder(View view) {
+        public EmployeeViewHolder(View view, RecyclerOnItemClickListener recyclerOnItemClickListener) {
             super(view);
             ButterKnife.bind(this, view);
+            this.recyclerOnItemClickListener = recyclerOnItemClickListener;
+            this.itemView.setOnClickListener(this);
+        }
+
+        @Override
```

Comment:
I saw a @OnClick from ButterKnife above. I think we should use the ButterKnife implementation instead of the native onClick